include/libtorrent/sha1_hash.hpp

```diff
@@ -262,7 +263,10 @@ namespace libtorrent
 		{
 			return std::string(reinterpret_cast<char const*>(&m_number[0]), size());
 		}
-
+		std::string to_hex() const
```

Comment:
I don't think this should be a member function. ( http://www.drdobbs.com/cpp/how-non-member-functions-improve-encapsu/184401197 )